src/fu-tool.c

```diff
@@ -5266,6 +5375,13 @@ main(int argc, char *argv[])
 			      /* TRANSLATORS: command description */
 			      _("Create an EFI boot entry"),
 			      fu_util_efiboot_create);
+	fu_util_cmd_array_add(cmd_array,
+			      "efiboot-hive",
```

Comment:
why is this called a `hive`?  It seems like it's basically just an `argument`.

Reply:
shim hive is the name of the specification, e.g. https://github.com/rhboot/shim/issues/678 -- although maybe tying it to the implementation here isn't wise. Perhaps efiboot-argument works, although that's a bit wordy. Maybe `efiboot-metadata` or `efiboot-store` work better?

Reply:
...on the other _other_ hand, hive is what Microsoft calls the thing that bcdedit sets -- i.e. we could use efiboot-hive on a bootmgfw.efi with a straight face.

Reply:
Got it; hive is fine then.